tests/scripts/format.py

```diff
@@ -111,19 +133,25 @@ def main(argv):
 
     bootstrapUrl = "https://raw.githubusercontent.com/dotnet/jitutils/master/" + bootstrapFilename
 
-    testfile.retrieve(bootstrapUrl, bootstrapFilename)
+    bootstrapPath = os.path.join(coreclr, bootstrapFilename)
+    testfile.retrieve(bootstrapUrl, bootstrapPath)
+
+    if not os.path.isfile(bootstrapPath):
+        print("Did not download bootstrap!")
+        sys.stdout.flush()
+        return -1
 
     # On Linux platforms, we need to make the bootstrap file executable
     if platform == 'Linux' or platform == 'OSX':
-        os.chmod(bootstrapFilename, 0751)
+        os.chmod(bootstrapPath, 0751)
 
     # Run bootstrap
     os.environ["PATH"] += os.pathsep + dotnetcliPath
-    proc = subprocess.Popen([os.path.join(coreclr, bootstrapFilename)], shell=True)
+    print(os.environ["PATH"])
```

Comment:
Is this just here for diagnostics, or do you want it in the log?  Doesn't matter to me; just curious.